pages/fiscal-sponsorship/mobile/index.js

```diff
@@ -0,0 +1,101 @@
+import { Box, Container, Heading } from 'theme-ui'
+import {
+  PillHolder,
+  AuthorPill,
+  DatePill
+} from '../../../components/announcements/pills'
+import Head from 'next/head'
+import NextLink from 'next/link'
+import styled from '@emotion/styled'
+import theme from '../../../lib/theme'
+import Meta from '@hackclub/meta'
+import Nav from '../../../components/nav'
+import ForceTheme from '../../../components/force-theme'
+import Footer from '../../../components/footer'
+import Copy from '../../../components/announcements/hcb-mobile.mdx'
+import SlackCTA from '../../../components/announcements/cta'
+import AnnouncementHolder from '../../../components/announcements/holder'
+import Balancer from 'react-wrap-balancer'
+
+const StyledLink = styled.a`
+  text-decoration: underline;
+  color: ${theme.colors.white};
+`
+
+const Link = props => {
+  const { href } = props
+  return (
+    <NextLink href={href} passHref>
+      <StyledLink>{props.children}</StyledLink>
+    </NextLink>
+  )
+}
+
+const RelonPage = () => (
+  <>
+    <Meta
+      as={Head}
+      title="HCB Mobile is here!"
+      description="View and manage your HCB organizations on the go. Issue cards, view transactions, and more!"
+      image="https://hc-cdn.hel1.your-objectstorage.com/s/v3/9f07613f2f1a8fa166d56695015fab73c0ab8060_og_image.png"
+    />
+    <ForceTheme theme="light" />
+    <Nav />
+    <Box
+      as="section"
+      sx={{
+        pt: [5, 6],
+        pb: [4, 5],
+        bg: 'rgb(104, 41, 205)',
+        backgroundImage: theme => theme.util.gx('purple', 'orange')
+      }}
+    >
+      <Container sx={{ textAlign: 'center', color: 'white' }}>
+        <Heading
+          as="h1"
+          variant="title"
+          sx={{
+            fontSize: [5, 6, null, 7],
+            span: {
+              WebkitTextStroke: 'currentColor',
+              WebkitTextStrokeWidth: ['2px', '3px'],
+              WebkitTextFillColor: 'transparent'
+            }
+          }}
+        >
+          <Link href="/fiscal-sponsorship">HCB Mobile</Link> is here!
+        </Heading>
+        <Balancer>
+          <Heading
+            as="h2"
+            variant="title"
+            sx={{
+              fontSize: [3, 4, 4, 4],
+              fontWeight: 400,
+              marginTop: '24px',
+              maxWidth: '900px',
+              mx: 'auto'
+            }}
+          >
+            View and manage your HCB organizations on the go. Issue cards, view transactions, and more!
```

Comment:
"View" twice here felt repetitive.
```suggestion
            Manage your HCB organizations on the go. Issue cards, view transactions, and more!
```